servers/physics/physics_server_sw.cpp

```diff
@@ -487,7 +487,8 @@ void PhysicsServerSW::body_set_space(RID p_body, RID p_space) {
 
 	while (body->get_constraint_map().size()) {
 		RID self = body->get_constraint_map().front()->key()->get_self();
-		ERR_CONTINUE(!self.is_valid());
+		if (!self.is_valid())
```

Comment:
This commit basically overrides 11701a46c0ee8cd635b6944c6363479b6c57ba5d, might be good to squash those two.